tests/MongoDB.Driver.Tests/Specifications/unified-test-format/UnifiedTestOperations/UnifiedAggregateDatabaseOperation.cs

```diff
@@ -0,0 +1,108 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+
+namespace MongoDB.Driver.Tests.Specifications.unified_test_format.UnifiedTestOperations
+{
+    public class UnifiedAggregateDatabaseOperation : IUnifiedTestOperation
+    {
+        private IMongoDatabase _database;
+        private AggregateOptions _options;
+        private PipelineDefinition<NoPipelineInput, BsonDocument> _pipeline;
+
+        public UnifiedAggregateDatabaseOperation(
+            IMongoDatabase database,
+            AggregateOptions options,
+            PipelineDefinition<NoPipelineInput, BsonDocument> pipeline)
```

Comment:
Standardize argument order: 

    database, pipeline, options

Why is there no session?